### PR TITLE
Add hammer tests

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

**Comment by ekohl** [comment]
Note I left part of it out since it wasn't used and somewhat ugly. https://community.theforeman.org/t/conversion-of-hammer-tests-in-systest-to-forklift/8093

File: roles/foreman_testing/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+- name: "Run bats tests"
+  include_role:
```
Comment (evgeni) [inline roles/foreman_testing/tasks/main.yaml]: I think you can also define `meta/main.yml` with
```
dependencies:
  - { role: 'bats', when: foreman_testing_bats }
  - { role: 'hammer_tests', when: foreman_testing_hammer_tests }
```

Not sure what is best, tho. (https://docs.ansible.com/ansible/latest/playbooks_reuse_roles.html says "You can conditionally execute a role. This is not generally recommended with the classic syntax, but is common when using `import_role` or `include_role`")

**Comment by evgeni** [comment]
Looks correct on first glance, will give it a few test rounds now.

**Comment by ekohl** [comment]
> The hammer tests are disabled on purpose? If so, ACK ;)

They are indeed since that matches the current behavior. They also need orgs and locs enabled which is not true on the default vanilla foreman.

**Pull request merged**